cirq/ops/linear_combinations_test.py

```diff
@@ -562,3 +562,20 @@ def test_linear_combination_of_operations_has_correct_pauli_expansion(
 ))
 def test_operation_expressions(expression, expected_result):
     assert_linear_combinations_are_equal(expression, expected_result)
+
+
+def test_pauli_sum_from_adding_strings():
+    q = cirq.LineQubit.range(2)
+    pstr1 = cirq.X(q[0]) * cirq.X(q[1])
+    pstr2 = cirq.Y(q[0]) * cirq.Y(q[1])
+    psum = pstr1 + pstr2
+    assert str(psum) == "1.000*X(0)*X(1)+1.000*Y(0)*Y(1)"
+    assert list(psum) == [pstr1, pstr2]
+
+
+def test_pauli_sum_from_list_of_strings():
+    q = cirq.LineQubit.range(2)
+    pstr1 = cirq.X(q[0]) * cirq.X(q[1])
+    pstr2 = cirq.Y(q[0]) * cirq.Y(q[1])
+    psum = cirq.PauliSum.from_pauli_strings([pstr1, pstr2])
+    assert str(psum) == "1.000*X(0)*X(1)+1.000*Y(0)*Y(1)"
```

Comment:
Need a bunch more tests. E.g. nothing is directly checking subtraction, formatting, iteration, raw initialization (though some of them happen to be hit indirectly as part of these tests).

Reply:
Added some more tests. Do you have any specific requests for more?